src/generators/go/renderers/StructRenderer.ts

```diff
@@ -21,18 +22,26 @@ type ${formattedName} struct {
 ${this.indent(this.renderBlock(content, 2))}
 }`;
   }
+  runFieldTagPreset(fieldName: string, field: CommonModel, type: FieldType = FieldType.field): Promise<string> {
+    return this.runPreset('fieldTag', { fieldName, field, type});
+  }
 }
-
 export const GO_DEFAULT_STRUCT_PRESET: StructPreset<StructRenderer> = {
   self({ renderer }) {
     return renderer.defaultSelf();
   },
-  field({ fieldName, field, renderer, type }) {
-    fieldName = renderer.nameField(fieldName, field);
+  async field({ fieldName, field, renderer, type }) {
+    const formattedFieldName = renderer.nameField(fieldName, field);
     let fieldType = renderer.renderType(field);
     if (type === FieldType.additionalProperty || type === FieldType.patternProperties) {
       fieldType = `map[string]${fieldType}`; 
     }
-    return `${ fieldName } ${ fieldType }`;
+    const fieldTags = await renderer.runFieldTagPreset(fieldName, field, type);
```

Comment:
I have problem with that line, why? If someone want to write `field` preset have to remember about running also `fieldTag`. I think that we should concatenate field and tag content inside `StructRenderer` rather than in preset. I hope you understand it :)

Reply:
On the other hand, nothing is stopping someone from using the field preset to also define tags 🤔 For example if you add the field preset and return:
    
```ts
return `${ fieldName } ${ fieldType } \`example:"name"\``;
```

Using any of the presets that add tags, wont work 🤷

Reply:
But as we discussed then you have the problem of overwriting previously overwritten tags. Only concatenation of these tags would somehow make sense if we were doing it inside the renderer. Then people would write the preset for the field and tags separately and then concatenate them inside the source code, here https://github.com/asyncapi/modelina/blob/10aca3b10d985596241761f8976697a72dc360e1/src/generators/go/GoRenderer.ts#L75

Reply:
More and more cases where AST would be better 😄